src/forms/FormFieldAddress/AddressDrawer/AddressDrawer.tsx

```diff
@@ -206,10 +206,10 @@ const AddressDrawer = (props: AddressDrawerProps): ReactElement => {
 
 		listOfAddresses.push({
 			id: id,
-			address1: state?.data?.address1,
-			address2: state?.data?.address2,
-			address3: state?.data?.address3,
-			city: state?.data?.city,
+			address1: state?.data?.address1.trim(),

```

Comment:
Reckon we should stick another null conditional on the address line properties? I just noticed if I try and "Save" the address draw with the required fields but without interacting with the address lines 1-3 it errors on those `undefined`s:

http://kube.simpleview.io:10000/?path=/story/formfields-formfieldaddress--playground

![image](https://github.com/simpleviewinc/sv-mosaic/assets/13272784/27cca011-f63a-4300-8fc9-960546281d21)

Reply:
@Porl91  Great shout. Checked in changes that employ optional chaining.